src/components/side-panel/layer-panel/dimension-scale-selector.tsx

```diff
@@ -35,7 +51,7 @@ const DimensionScaleSelector = ({label, onSelect, options, scaleType, disabled =
       </PanelLabel>
       <ItemSelector
         disabled={disabled}
-        selectedItems={scaleType}
+        selectedItems={scaleType ? [scaleType] : []}
```

Comment:
shouldn't need to convert to array. `selectedItems` can be a single object or an array of objects

Reply:
fixed